src/Classes/ServiceAPI/MyRadio_Timeslot.php

```diff
@@ -1261,7 +1261,7 @@ public function signInGuests(string $guestInfo, $locationid)
         self::$db->query(
             'INSERT INTO sis2.guest_signin (show_season_timeslot_id, signerid, location, guest_info)
                 VALUES ($1, $2, $3, $4)',
-            [$this->getID(), MyRadio_User::getInstance()->getID(), $locationid, $guestInfo]
+            [$this->getID(), MyRadio_User::getInstance()->getID(), $locationid, htmlspecialchars($guestInfo, ENT_QUOTES)]
```

Comment:
Screm - I'd rather put in the cleanest data possible, and format it when you display it. (Also, Twig might take care of escaping itself?)

Reply:
Wait, you use `html`, nvm - either way, move `htmlspecialchars` to Scheduler/tracking.php

Reply:
nah, this seems to be the only way which works, remember say on quotes (rip) and show descriptions you can xss